.eslintignore

```diff
@@ -0,0 +1,2 @@
+# Builded files
+**/lib/*.js
```

Comment:
@leerob we should either move the `--ignore-pattern` from the `lint:js` script in `package.json` in here or remove this file and leave the ignores in `package.json`. Thoughts?